backend/corpora/common/corpora_orm.py

```diff
@@ -239,6 +239,7 @@ class DbCollection(Base, AuditMixin, TimestampMixin):
     data_submission_policy_version = Column(StrippedString, nullable=True)
     tombstone = Column(Boolean, default=False, nullable=False)
     publisher_metadata = Column(JSON, nullable=True)
+    revision_of = Column(String, nullable=True)
```

Comment:
There should be a foreign key constraint that this a key in the project table. https://docs.sqlalchemy.org/en/14/orm/basic_relationships.html#one-to-one